src/lib/viewers/media/MediaBaseViewer.js

```diff
@@ -212,6 +208,40 @@ class MediaBaseViewer extends BaseViewer {
     }
 
     /**
+     * Handler for autoplay
+     *
+     * @private
+     * @emits autoplay
+     * @return {void}
+     */
+    handleAutoplay() {
+        const isAutoplayEnabled =
+            this.cache.has(MEDIA_AUTOPLAY_CACHE_KEY) && this.cache.get(MEDIA_AUTOPLAY_CACHE_KEY) === 'Enabled';
```

Comment:
I don't think you have to do the first check. `get()` in Cache.js already does a check